api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -655,17 +659,31 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     dbWorkspace.setCdrVersion(fromWorkspace.getCdrVersion());
```

Comment:
I thought you were able to override the CDR version of the workspace when cloning -- you said you'd tested this -- but this logic doesn't seem to allow for it. Am I missing the part where you change it?

Note that if you *do* change the CDR version to a version with a more restrictive data access level, ideally we would kick out any collaborators that don't have that data access level... they won't be able to do anything with it. (We should ideally warn the user about this if they request to clone collaborators and this happens.)

Reply:
We should probably file an issue on the second point then so we don't forget about it.